Resources/doc/INSTALL.md

```diff
@@ -22,7 +22,7 @@ Add the following to your composer.json and run `php composer.phar update netgen
 
 ### Activate the bundle
 
-Activate the bundle in `ezpublish\EzPublishKernel.php` file.
+Activate the bundle in `ezpublish/EzPublishKernel.php` file.
```

Comment:
You will need to rebase the pull request against the master to remove this diff, since it's already merged from your previous commit.